src/js/tech/html5.js

```diff
@@ -962,6 +962,26 @@ Html5.canControlVolume = function() {
   }
 };
 
+/**
+ * Check if the volume can be muted in this browser/device.
+ * Some devices, e.g. iOS, don't allow chaning volume
+ * but permits muting/unmuting.
+ *
+ * @return {bolean}
+ *      - True if volume can be muted
+ *      - False otherwise
+ */
+Html5.canMuteVolume = function() {
+  try {
+    const muted = Html5.TEST_VID.muted;
+
+    Html5.TEST_VID.muted = !muted;
```

Comment:
I think we had some bugs where iOS didn't always mute the video with just the `muted` property but also required setting the attribute directly. We probably wanted to do both.